core/capabilities/ccip/ocrimpls/contract_transmitter_test.go

```diff
@@ -316,7 +317,7 @@ func newTestUniverse(t *testing.T, ks *keyringsAndSigners[[]byte]) *testUniverse
 		contractName,
 		methodTransmitWithSignatures,
 		ocr3HelperAddr.Hex(),
-		ocrimpls.ToCommitCalldata,
+		ocrimpls.NewToCommitCalldata(consts.MethodCommit, ""),
```

Comment:
Can we have a test for the Solana transmission?

Reply:
The Solana e2e test would cover this. I'm not sure how a unit test would work.